config/database.yml

```diff
@@ -20,6 +20,8 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  user: corcovado
+  password: carambola4You!
```

Comment:
@cclics Hey Carla, I'd suggest reverting this file for this push and changing your password. GH logs are forever, so please consider this password compromised and change it if it used with any other online/system accounts. 

You could keep these changes in your local app and prevent it from being pushed to GH by:
1.  Manually confirming that it isn't added to your commits by checking `git status` before each commit
2. Adding this file to a `.gitignore` file to automatically avoid committing changes

Happy to hop on a call or record something to go over this process

Reply:
@ncbui Hey Cam, thanks for review!  I got the original version from database.yml with the command checkout now I can use git restore -stage or commit. But I also noticed the Gemfile and Gemfile.lock are on branch main updated by me.
It would be great if we can have a quick call/chat for this.

Reply:
Hi Cam @ncbui, 
-database.yml  file is reverted to original file. 
-Gem file as well file is reverted to original file.

thank you for the hint about `.gitignore`